src/css/application.css

```diff
@@ -156,11 +154,17 @@ body {
   height: 2em;
   border-radius: 50%;
   vertical-align: middle;
-  line-height: 1em;
+  margin-right: .5em;
+}
+
+.top-bar__group {
+  display: flex;
+  flex: 0 0 auto;
+  width: 50%;
```

Comment:
Why fix the width at 50%? Seems like we’re losing most of the benefit of having this be a flex item… why not just do a `1 0 auto`?